legacy/src/main/java/org/opensearch/sql/legacy/pit/PointInTimeHandler.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.legacy.pit;
+
+import static org.opensearch.sql.common.setting.Settings.Key.SQL_CURSOR_KEEP_ALIVE;
+
+import lombok.Getter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.search.CreatePitRequest;
+import org.opensearch.action.search.CreatePitResponse;
+import org.opensearch.action.search.DeletePitRequest;
+import org.opensearch.action.search.DeletePitResponse;
+import org.opensearch.client.Client;
+import org.opensearch.core.action.ActionListener;
+import org.opensearch.sql.legacy.esdomain.LocalClusterState;
+
+/** Handler for Point In Time */
+public class PointInTimeHandler implements PIT {
```

Comment:
Can we name PointInTimeHandler and PointInTimeHandlerImpl?

Reply:
done